src/Components/ColorPalette/ColorPalette.jsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from "react";
+import "./colorPalette.css";
+
+const colorsInArray = [
+	"#f28983",
+	"#fbbc04",
+	"#FFF475",
+	"#CCFF90",
+	"#A7FFEB",
+	"#CBF0F8",
+	"#AECBFA",
+	"#D7AEFB",
+	"#FDCFE8",
+	"#E6C9A8",
+	"#E8EAED",
+	"#238AC5",
+];
+
+
+export const ColorPalette = ({ setInputCardDetails, inputCardDetails }) => {
+	const [isHidden, setHidden] = useState(true);
+	return (
+		<>
+			<div
+				onMouseOver={() => {
```

Comment:
Try to format the code, can use Prettier for that.

Reply:
Sure will do it for sure ;-)